src/main/java/fhirspark/FhirSpark.java

```diff
@@ -216,10 +216,16 @@ public static void main(final String[] args) throws Exception {
      */
     private static boolean validateRequest(Request req) {
         String portalDomain = settings.getPortalUrl();
-        String validatePath = "api/studies/" + settings.getMtbStudy() + "/patients/"
```

Comment:
The parameter `settings.getMtbStudy()` is not used anymore, so why don't you remove it?